### ISSUE TITLE
openeconomics.net frontpage is 1.3MB (!)

### ISSUE THREAD

**Issue by rufuspollock** [okfn/okfn.org#87]
Also takes ~3s to load. See http://tools.pingdom.com/fpt/#!/knsQZApWf/openeconomics.net

Some of this is the very elaborate front page image (~0.5Mb ...)

/cc @vndimitrova @mintcanary

**Comment by smth** [comment]
Yeah, there are a couple of unnecessarily large image files there :/ That aside, it's probably an average size for a web page isn't it?

**Comment by rufuspollock** [comment]
I'm not sure but my feeling is that over 1MB is pretty big  - think about what this is like for mobile or in a country with slow internet. (Perhaps worth doing some quick research)

I know some of this is images for blog posts which we can't control but for the rest we should try and pull this down ...
